<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flashcard</class>
 <widget class="QWidget" name="Flashcard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>380</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedOuter">
     <widget class="QWidget" name="edit_view">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="edits"/>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}

</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="card_view">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedInner">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="FCFront" name="front_side"/>
         <widget class="FCBack" name="back_side"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottom_btn_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="edit_card_btn">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}

</string>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="card_index">
           <property name="styleSheet">
            <string notr="true">color: black;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="flip_card_btn">
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgba(255, 98, 71, 0.6);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(255, 98, 71, 1);
}

</string>
           </property>
           <property name="text">
            <string>Flip</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FCFront</class>
   <extends>QWidget</extends>
   <header>Flashcards/fcfront.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FCBack</class>
   <extends>QWidget</extends>
   <header>Flashcards/fcback.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
